docs/domains/index.html

```diff
@@ -1404,6 +1407,41 @@
       "type": "object",
       "properties": {}
     },
+    "Model": {
+      "type": "object",
+      "properties": {
+        "encryptionSecret": {
+          "description": "WDT encryption key passphrase secret.",
+          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
+        },
+        "domainType": {
+          "description": "WDT domain type: Legal values: WLS, RestrictedJRF, JRF",
+          "type": "string",
+          "enum": [
+            "WLS",
+            "RestrictedJRF",
+            "JRF"
+          ]
+        },
+        "configMapName": {
```

Comment:
configMapName --> configMap?   This would match the naming pattern used for other attributes like 'keyWalletConfigMap' and 'overridesConfigMap'.